packages/wallet/ui/src/views/Dapps.jsx

```diff
@@ -23,7 +23,8 @@ export const DappsWithoutContext = ({ dapps }) => {
     const handleClick = event => {
       setAnchorEl(event.currentTarget);
     };
-    const handleClose = () => {
+    const handleClose = event => {
+      event.stopPropagation();
```

Comment:
Was this causing a bug? Just wondering because it doesn't seem related to this change

Reply:
Ah, that was a mistake.  I'll take that out.